octobot_trading/exchanges/config/ccxt_exchange_settings.py

```diff
@@ -0,0 +1,174 @@
+import octobot_trading.exchanges.config.exchange_test_status as exchange_test_status
+import octobot_trading.exchanges.config.generic_exchange_settings as generic_exchange_settings
+import octobot_trading.exchanges.parser as parser
+
+
+class CCXTExchangeConfig(generic_exchange_settings.GenericExchangeConfig):
+    """
```

Comment:
Same as all ccxt related stuff. It should be put in connectors package. We may need to create a ccxt subpackage in conntector

Reply:
i moved it